sources/editor/Xenko.Core.Assets.Editor/Components/TemplateDescriptions/ViewModels/ExistingProjectViewModel.cs

```diff
@@ -2,21 +2,29 @@
 // Distributed under the MIT license. See the LICENSE.md file in the project root for more information.
 using System;
 using System.Collections.Generic;
+using System.Diagnostics;
 using System.Linq;
 using System.Windows.Media.Imaging;
+using Xenko.Core.Assets.Editor.ViewModel;
 using Xenko.Core.Assets.Templates;
 using Xenko.Core.IO;
+using Xenko.Core.Presentation.Commands;
 using Xenko.Core.Presentation.ViewModel;
 
 namespace Xenko.Core.Assets.Editor.Components.TemplateDescriptions.ViewModels
 {
     public class ExistingProjectViewModel : DispatcherViewModel, ITemplateDescriptionViewModel
     {
-        public ExistingProjectViewModel(IViewModelServiceProvider serviceProvider, UFile path)
+        private Action<ExistingProjectViewModel> RemoveCallBack { get; }
+
+        public ExistingProjectViewModel(IViewModelServiceProvider serviceProvider, UFile path, Action<ExistingProjectViewModel> removeCallBack)
```

Comment:
I don't really like the word "callback", especially since it is here a different concept than C/C++ callbacks. Rather use the word "action".